deploy_project.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -e
+
+# Function to display error message and exit
+error_exit() {
+    echo "$1" >&2
+    exit 1
+}
+
+# Build the release bundle
+echo "Building the release bundle..."
+make release
+ZIP_PATH="./EmpowerPlant_release.zip"
+
+# Check if gh is installed
+if ! command -v gh &> /dev/null; then
+  error_exit "gh is not installed, make sure you run 'make init' (see README.md)."
+fi
```

Comment:
I wouldn't even do this, it's handled by `brew bundle` and we can just let the `gh` invocation on line 41 fail the script if `make init` hasn't run yet.

Reply:
hm, not sure I follow.  For `brew bundle` to run we need to run `make init` manually, right? (since `release` target doesn't depend on `init` target). That's why I thought we should call out running `make init` here.